src/Tokenizer/Transformer/ArraySquareBraceTransformer.php

```diff
@@ -40,6 +40,16 @@ public function getCustomTokenNames()
     /**
      * {@inheritdoc}
      */
+    public function getRequiredPhpVersionId()
+    {
+        // short array syntax was introduced in PHP 5.4, but the fixer is smart
+        // enough to handel it even before 5.4
+        return 50000;
```

Comment:
that was my first implementation. then i throw it away.
why? because then it would be so easy to ever overwrite it and always apply all transformers, even when not needed. and waste time...